apps/qup-web/postcss.config.cjs

```
@@ -0,0 +1,6 @@
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)